erf.go

```diff
@@ -0,0 +1,124 @@
+// Copyright ©2017 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package mathext
+
+import "math"
+
+// NormalQuantile computes the quantile function (inverse CDF) of the standard
+// normal.
+func NormalQuantile(p float64) float64 {
+	switch {
+	case p > 1.0 || p < 0.0:
```

Comment:
I think this may be my least favorite thing about coding in Go. Not too shabby for the language.